### PR TITLE
New package: cuda-7.0.28

### PR DESCRIPTION
**Pull request by tonkazoid**

### CODE REVIEW

File: srcpkgs/cuda/template [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	    -e 's|ParallelComputing|ParallelComputing;|g' \
+	    -i pkg/install-linux.pl
+}
+
+do_install() {
+	cd pkg
+	perl install-linux.pl -prefix="${DESTDIR}/opt/cuda" -noprompt -nosymlink
+	perl install-sdk-linux.pl -cudaprefix="${DESTDIR}/opt/cuda" -prefix="${DESTDIR}/opt/cuda/samples" -noprompt
+	sed -i "/unsupported GNU/d" ${DESTDIR}/opt/cuda/include/host_config.h
+	vinstall ${FILESDIR}/cuda.sh 755 ${DESTDIR}/etc/profile.d
```
Comment (xtraeme) [inline srcpkgs/cuda/template]: cuda.sh does not need to be 755, change it to 644 please.

**Comment by xtraeme** [comment]
This one cannot be applied.

**Comment by tonkazoid** [comment]
Hmm, because of the size? pkg directory uncompressed is ~2.14GB.

**Comment by xtraeme** [comment]
I mean the pull request, your fork might not be fully uptodate compared to origin.

**Comment by xtraeme** [comment]
Please rebase to make this applicable.

**Pull request closed without merging**